--- ./node_modules/ajv/lib/compile/index.js	2022-11-22 00:46:14.706367384 +0000
+++ ./node_modules/ajv/lib/compile/index.js	2022-11-24 18:37:08.837573837 +0000
@@ -1,5 +1,19 @@
 'use strict';
 
+/*
+ * Some solicitous environments expose a blocked Function constructor.
+ * Fortunately, the untinkered Function is leaking in some of them,
+ * so let's use it:
+ */
+var Function_ = Function;
+try {
+  new Function_();
+} catch (e) {
+  if (!([] instanceof Object)) {
+    Function_ = globalThis.global.JSON.parse("[]").constructor.prototype.map.constructor;
+  }
+}
+
 var resolve = require('./resolve')
   , util = require('./util')
   , errorClasses = require('./error_classes')
@@ -117,7 +131,7 @@
     // console.log('\n\n\n *** \n', JSON.stringify(sourceCode));
     var validate;
     try {
-      var makeValidate = new Function(
+      var makeValidate = new Function_(
         'self',
         'RULES',
         'formats',
